src/styles/utilits/_reset.scss

```diff
@@ -0,0 +1,12 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
```

Comment:
what'll changed if you remove padding here?

Reply:
Nothing 😃 But it's defalut style for my "reset.css" file.